easybuild/easyblocks/c/cp2k.py

```diff
@@ -314,7 +318,7 @@ def configure_common(self):
                                                        self.cfg['extracflags']),
             'DFLAGS': ' -D__parallel -D__BLACS -D__SCALAPACK -D__FFTSG %s' % self.cfg['extradflags'],
 
-            'LIBS': os.getenv('LIBS'),
+            'LIBS': os.getenv('LIBS', ''),
```

Comment:
LIBS may not be set, FFT_INC_DIR should always be set

but I can add a proper default there too, thanks for the suggestion